iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/SasTokenTools.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package tests.integration.com.microsoft.azure.sdk.iot.helpers;
+
+/**
+ * Tools class for SAS token helper functions
+ */
+public class SasTokenTools
+{
+    private static final String KEY_VALUE_PAIR_SEPARATOR = "&";
+    private static final String EXPIRY_TIME_KEY = KEY_VALUE_PAIR_SEPARATOR + "se=";
+
+    /**
+     * Takes in a valid SAS token and returns an expired version of that same SAS token. This function works for both
+     * device and service SAS tokens.
+     * @param sasToken a valid SAS token to make expired.
+     * @return an expired SAS token.
+     */
+    public static String makeSasTokenExpired(String sasToken)
```

Comment:
Our library's public SAS token classes don't allow you to create a token that expires before the current moment, so I added this function in our test code instead. It really should only be used in test scenarios like token renewal tests